content/strategy-goals/strategy/code-insights/index.md

```diff
@@ -40,113 +41,109 @@ To deliver on our mission, we will:
-- If building a feature that might be reused across the Sourcegraph platform, Code Insights consults with other product engineering teams
+- We make automated alerting for Insights and related features first-class priorities
 
-**What this does _not_ look like:**
+**What this does not look like:**
 
-- Code Insights won't avoid introducing net new features if no other Sourcegraph feature has yet built them
+- We don't spam developers with unnecessary information or alerts. We make it easy to configure alerts that you actually care about.
 
-### Code Insights is a product _with graphs_, not a _graphing product_
+### For Engineering Leaders
 
-The primary value of Code Insights comes from the questions it can answer for its users in service of [Sourcegraph use cases](../index.md#use-cases). While Code Insights will continue to add graph UI features as customers demonstrate need and value –- or even just desire -- we will primarily focus on building new insight types or graph flows (such as monitoring, cross-feature integrations, and filtering/drilldown ability) rather than new chart types or especially fancy charting customizations.
+#### Code Insights was built first for engineering leaders
 
-**What this looks like:**
+The answer to "why did we build Code Insights ahead of other cool projects Sourcegraph can pursue?" is that Code Insights serves the needs of engineering leaders.
 
-- Even in rarer cases when we don't have user feedback to differentiate the priority of two features, we'll prioritize features that let you answer more types of questions, or answer them more specifically, rather than features that let you change the graph display for aesthetic purposes
-- We sometimes delay "standard" graphing UI features, such as persisted re-orderable dashboard displays, because they won't best unlock value for our users relative to other work
+Engineer leaders may not need to use Sourcegraph directly for code search (though their teams do), but they do need to answer questions about their code that Sourcegraph, via Code Insights, is well-positioned to serve.
 
-**What this does not look like:**
+The below principles are an abbreviation of what we use when building for engineering leaders. For full details, see [older revisions of this page](https://sourcegraph.com/github.com/sourcegraph/handbook/-/commit/c45cd7e2e8d54f2090f79417414dd0e131acc2b0?visible=2). These principles will become relevant again after we build Insights features that serve the core developer
 
-- Code Insights will still prioritize adding new chart types if they _do_ enable insights to answer new questions, such as a heatmap chart type to visualize changes in a non-Cartesian coordinates way.
+- Code Insights layers together the Sourcegraph feature ecosystem
+- Code Insights is NOT for tracking individual developers
+- Code Insights is an enterprise quality professional tool
```

Comment:
```suggestion
- Code Insights is an enterprise-quality professional tool that helps leaders make strategic decisions and track key initiatives
```

Reply:
@AlicjaSuska I'm ignoring/not making this change since the original point here is about the principle of quality and what that means (regardless of what it is used for) but I appreciate the note.